### ISSUE TITLE
Filebeat: run a command at an interval and consume stdout

### ISSUE THREAD

**Issue by fledder** [elastic/beats#14638]
I have a particular piece of software (Wonderware InTouch, an industrial control / SCADA platform) that generates logs in a binary format that cannot be consumed directly. There is a GitHub project that provides a command line utility that will interpret the logs and output to stdout. If there was a way to configure the stdin input in Filebeat to periodically run the utility and consume whatever it sends to stdout, that would make it much easier to integrate these logs.

So in summary, this request is for the Filebeat (or other appropriate beat) to be configured with a shell command and an interval. On the interval, the beat runs the command and pipes the command's stdout to the input of the beat and forwards that on to Elasticsearch / Logstash.

Thanks!

**Comment by greenmoss** [comment]
I stumbled across this ticket while looking for the same thing. I am trying out execbeat, FWIW: https://github.com/christiangalsterer/execbeat
